### ISSUE TITLE
Savon.expects action namespacing issue

### ISSUE THREAD

**Issue by BillWatts** [savonrb/savon_spec#18]
When doing savon.expects on actions that are namespaced the tests looks for the namespaced attribute rather the action.

Rspec test.

```
it "has private method login which which makes soap call and returns token" do
  stub_request(:get, "http://wcf.emarketingsaas.com/Sys.svc?wsdl").to_return(:status => 200, :body => "", :headers => {})
  savon.expects("Login").returns("../../epicore_email_service/login/login_response.xml")
 @class.send(:login).should == "eb910988-2820-4dbe-aca6-1f9610bfbf8c"
end
```

Rspec output.

```
Failure/Error: @class.send(:login).should == "eb910988-2820-4dbe-aca6-1f9610bfbf8c"
 Savon::Spec::ExpectationError:
   expected :Login to be called, got: {"xmlns"=>"http://wcf.spinnakerpro.net/1.0"}
 # /Users/BillWatts/Sites/ubn-webservices/services/epicore_email_service.rb:126:in `login'
 # ./epicore_email_service_spec.rb:36:in `block (2 levels) in <top (required)>'
```

Example of how the request is set up.

```
soap_client = Savon::Client.new do
  wsdl.endpoint = _self.instance_variable_get(:@credentials)["login_wsdl"]
  wsdl.namespace = "http://wcf.spinnakerpro.net/1.0/Sys/Login"
end

response = soap_client.request "Login" do
  http.headers['soapAction'] = "http://wcf.spinnakerpro.net/1.0/Sys/Login"
  soap.input = ["Login", {"xmlns" => "http://wcf.spinnakerpro.net/1.0"}]
  soap.body = {
    "Organization" => _self.instance_variable_get(:@credentials)["org"],
    "User" => _self.instance_variable_get(:@credentials)["username"],
    "Password" => _self.instance_variable_get(:@credentials)["password"]
  }
end
```

Example of how Savon renders the xml.

```
<?xml version="1.0" encoding="UTF-8"?>
<env:Envelope xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:wsdl="http://wcf.spinnakerpro.net/1.0" xmlns:env="http://schemas.xmlsoap.org/soap/envelope/">
  <env:Body>
    <Login xmlns="http://wcf.spinnakerpro.net/1.0">
      <Organization></Organization>
      <User></User>
      <Password></Password>
    </Login>
  </env:Body>
</env:Envelope>
```

I think this is a bug, but it could be poor wsdl design on the remote end or something I'm doing incorrectly.

**Comment by rubiii** [comment]
this will be fixed with savon 2.0. please [read the documentation](http://savonrb.com/version2.html) and give it a try if you can.

**Comment by BillWatts** [comment]
I've tried this with 2.0 and all is well.  Thanks.
